Core.DomainModel/Organization/PaymentRegistrationDetails.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using Core.DomainModel.ItContract;
+
+namespace Core.DomainModel.Organization
+{
+    public class PaymentRegistrationDetails
+    {
+        public PaymentRegistrationDetails(int unitId, ItContract.ItContract itContract)
+        {
+            ItContract = itContract;
+            InternalPayments = itContract.GetInternalPaymentsForUnit(unitId);
```

Comment:
seems wierd that you call these two methods here when the calling code has already extracted both external and internal payments.. consider if you should pass the intern and extern payments in the ctor in stead